executor/aggfuncs/func_max_min.go

```diff
@@ -255,13 +255,14 @@ type maxMin4Decimal struct {
 
 func (e *maxMin4Decimal) AllocPartialResult() PartialResult {
 	p := new(partialResult4MaxMinDecimal)
+	p.val = *new(types.MyDecimal)
```

Comment:
This operation can be removed? since we only set it once the first non-NULL value comes.